src/Main.java

```diff
@@ -1,5 +1,317 @@
+        //    +
+        //    Please enter integer:
+        //    5
+        //    Which operation would you like to use: *,/,+,-,=?
+        //    -
+        //    Please enter integer:
+        //    3
+        //    Which operation would you like to use: *,/,+,-,=?
+        //    *
+        //    please enter integer
+        //    5
+        //    Which operation would you like to use: *,/,+,-,=?
+        //    =
+        //    Calculator Result is 10;
+        System.out.println("\n");
+        System.out.println("Task 1.f");
+        resultOfOperation = 0;
+        enteredNumber = 0;
+        operator = "";
+
+        while (!operator.equals("=")) {
+            System.out.println("Which operation would you like to use: *,/,+,-,=");
+            operator = reader.next();
+
+            if (operator.equals("="))
+                break;
+
+            System.out.println("Please enter integer: ");
+            enteredNumber = reader.nextInt();
+
+            switch (operator) {
+                case "+":
+                    resultOfOperation += enteredNumber;
+                    break;
+                case "-":
+                    resultOfOperation -= enteredNumber;
+                    break;
+                case "*":
+                    resultOfOperation *= enteredNumber;
+                    break;
+                case "/":
+                    resultOfOperation /= enteredNumber;
+                    break;
+            }
+        }
+        System.out.println("Calculator result is: " + resultOfOperation);
+
+        //2. Napravite petlju u koju unosite brojeve. Kada petlja detektuje broj -10,
+        // ispise vam se niz brojeva koje ste unjeli.
+        //
+        // Zatim ce vam se na ekranu pojaviti pitanje da li zelite da prosirite array sa opcijama y/n.
+        //Ukoliko upisete y, ponovno unosite brojeve koji cete dodavati na niz
+        // kada ponovno unesete -10 ispisu vam se svi brojevi koje ste unjeli.
+
+        System.out.println("\n");
+        System.out.println("Task 2");
+        List<Integer> listOfNumbers = new ArrayList<Integer>();
+        int numberOfArray = 0;
+        while (numberOfArray != -10) {
+            System.out.println("Enter the number of array: ");
+            numberOfArray = reader.nextInt();
+            if (numberOfArray != -10)
+                listOfNumbers.add(numberOfArray);
+        }
+
+        System.out.println("Numbers of array are: ");
+        System.out.println(listOfNumbers);
+
+        System.out.println("Would you like to add more numbers into array ? Y / N");
+        String addMoreItems = reader.next();
+        if (addMoreItems.equals("Y")) {
+            numberOfArray = 0;
+            while (numberOfArray != -10) {
+                System.out.println("Add new number to existing array: ");
+                numberOfArray = reader.nextInt();
+                if (numberOfArray != -10)
+                    listOfNumbers.add(numberOfArray);
```

Comment:
again this solution was not to be used, we didnt work with lists , and the effort was neccessary to help your problem solving skills.